.bazelci/ingest_grpc_logs.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python2
+import json
+import time
+import sys
+from dateutil import parser
+from collections import defaultdict
+
+REMOTE_EXECUTIONS = "build.bazel.remote.execution.v2.Execution/Execute"
+CAS_UPLOAD = "google.bytestream.ByteStream/Write"
+CAS_DOWNLOADS = "google.bytestream.ByteStream/Read"
+ACTION_CACHE_REQUESTS = ("build.bazel.remote.execution.v2.ActionCache/"
+                         "GetActionResult")
+
+
+def main():
+    n = len(sys.argv)
+    for i in range(1, n):
+        print("=====================")
+        print("Build #%d results" % i)
+        methods = defaultdict(list)
+        parsed = parse_file(sys.argv[i])
+        count_remote_actions(parsed, methods)
+        calculate_latency(methods)
```

Comment:
looks good.  
For the CI can we also check that the 2nd grpc log makes 0 remote execution requests and 0 CAS uploads.  If the second log makes those requests it would mean something is wrong with our caching and we have a regression.  If the 2nd log makes those requests we can have the script fail and then hopefully the CI plan fail too.  

something like:  
```
if i == 2:
    ensure_fully_cached_build(parsed, methods)
```
Or whatever you think is cleaner

Reply:
Hi, Trevor. It already does that. It will print out build:1 and build:2 results which include what you've described. Or you are talking for explicit check that the second run we for sure get 0 remote execution.

Reply:
Yeah, I'd like to gate on the 2nd log's results it to avoid regressions. I'm going to refactor the ActionCache and CAS in the future and want to make sure they preserve 100% of cache under this circumstance in the CI.

Reply:
Printing out the build:1 build:2 results is still helpful and will probably be used by clients.  We can glance at it in the CI as well.  But I want the CI step to fail if we are rebuilding when we shouldn't.